diff -urN a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2012-07-01 15:32:22.655369000 -0500
+++ b/CMakeLists.txt	2012-07-01 16:46:02.021369000 -0500
@@ -131,7 +131,7 @@
 
 install (
     FILES ${CMAKE_CURRENT_BINARY_DIR}/libcompizconfig.pc
-    DESTINATION ${COMPIZ_DESTDIR}${libdir}/pkgconfig
+    DESTINATION ${libdir}/pkgconfig
 )
 
 
@@ -160,8 +160,8 @@
 endif (NOT _find_compizconfig)
     add_custom_target (
         findcompizconfig_install
-	${CMAKE_COMMAND} -E make_directory ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules &&
-	${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/cmake/FindCompizConfig.cmake ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules
+	${CMAKE_COMMAND} -E make_directory ${CMAKE_ROOT}/Modules &&
+	${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/cmake/FindCompizConfig.cmake ${CMAKE_ROOT}/Modules
     )
 
 macro (add_uninstall)
diff -urN a/cmake/CMakeLists.txt b/cmake/CMakeLists.txt
--- a/cmake/CMakeLists.txt	2012-07-01 15:32:22.654369000 -0500
+++ b/cmake/CMakeLists.txt	2012-07-01 16:46:07.490369000 -0500
@@ -4,5 +4,5 @@
 
 install (
     FILES ${_files}
-    DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/compiz/cmake
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/compiz/cmake
 )
diff -urN a/cmake/LibCompizConfigCommon.cmake b/cmake/LibCompizConfigCommon.cmake
--- a/cmake/LibCompizConfigCommon.cmake	2012-07-01 15:32:22.654369000 -0500
+++ b/cmake/LibCompizConfigCommon.cmake	2012-07-01 16:46:05.042369000 -0500
@@ -122,7 +122,7 @@
 
     install (
 	TARGETS ${bname}
-	DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/lib/compizconfig/backends
+	DESTINATION ${CMAKE_INSTALL_PREFIX}/lib/compizconfig/backends
     )
 
     if (NOT _COMPIZCONFIG_INTERNAL)
diff -urN a/config/CMakeLists.txt b/config/CMakeLists.txt
--- a/config/CMakeLists.txt	2012-07-01 15:32:22.655369000 -0500
+++ b/config/CMakeLists.txt	2012-07-01 16:46:09.761369000 -0500
@@ -1,10 +1,10 @@
 set (_config_file
      config)
 
-compiz_opt_install_file (${_config_file} ${COMPIZ_SYSCONFDIR}/compizconfig/${_config_file})
+ compiz_opt_install_file (${_config_file} ${COMPIZ_SYSCONFDIR}/compizconfig/${_config_file})
 
 add_custom_target (
     config_install
-    ${CMAKE_COMMAND} -E make_directory ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules &&
-    ${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/config/${_config_file} ${COMPIZ_DESTDIR}${COMPIZ_SYSCONFDIR}/compizconfig/${_config_file}
+    ${CMAKE_COMMAND} -E make_directory ${CMAKE_ROOT}/Modules &&
+    ${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/config/${_config_file} ${COMPIZ_SYSCONFDIR}/compizconfig/${_config_file}
     )
diff -urN a/include/CMakeLists.txt b/include/CMakeLists.txt
--- a/include/CMakeLists.txt	2012-07-01 15:32:22.656369000 -0500
+++ b/include/CMakeLists.txt	2012-07-01 16:46:11.785369000 -0500
@@ -5,5 +5,5 @@
 
 install (
 	FILES ${_headers}
-	DESTINATION ${COMPIZ_DESTDIR}${includedir}
+	DESTINATION ${includedir}
 	)
diff -urN a/src/CMakeLists.txt b/src/CMakeLists.txt
--- a/src/CMakeLists.txt	2012-07-01 15:32:22.654369000 -0500
+++ b/src/CMakeLists.txt	2012-07-01 16:46:16.042369000 -0500
@@ -76,5 +76,5 @@
 
 install (
     TARGETS compizconfig
-    DESTINATION ${COMPIZ_DESTDIR}${libdir}
+    DESTINATION ${libdir}
 )
