diff -urN a/kernel/nv-drm.c b/kernel/nv-drm.c
--- a/kernel/nv-drm.c	2013-08-13 19:33:23.000000000 -0700
+++ b/kernel/nv-drm.c	2014-03-24 14:41:52.319326907 -0700
@@ -106,7 +106,6 @@
     .unlocked_ioctl = drm_ioctl,
     .mmap = drm_gem_mmap,
     .poll = drm_poll,
-    .fasync = drm_fasync,
     .read = drm_read,
     .llseek = noop_llseek,
 };
@@ -202,12 +201,7 @@
     memset(&nv_obj->base, 0, sizeof(nv_obj->base));
     nv_obj->pages = pages;
 
-    ret = drm_gem_private_object_init(nvl->drm, &nv_obj->base, size);
-    if (ret)
-    {
-        status = RM_ERR_OPERATING_SYSTEM;
-        goto done;
-    }
+    drm_gem_private_object_init(nvl->drm, &nv_obj->base, size);
 
     ret = drm_gem_handle_create(file_priv, &nv_obj->base, handle);
     if (ret)
diff -urN a/kernel/nv-linux.h b/kernel/nv-linux.h
--- a/kernel/nv-linux.h	2013-08-13 19:33:23.000000000 -0700
+++ b/kernel/nv-linux.h	2014-03-24 14:45:21.368423085 -0700
@@ -331,6 +331,9 @@
 #endif
 
 #if defined(NV_LINUX_ACPI_EVENTS_SUPPORTED)
+#if LINUX_VERSION_CODE >= KERNEL_VERSION(3, 13, 0)
+#define NV_ACPI_OS_WAIT_EVENTS_COMPLETE()
+#else
 #if defined(KERNEL_2_4) || \
   (NV_ACPI_OS_WAIT_EVENTS_COMPLETE_ARGUMENT_COUNT == 1)
 #define NV_ACPI_OS_WAIT_EVENTS_COMPLETE() \
@@ -342,6 +345,7 @@
 #error "NV_ACPI_OS_WAIT_EVENTS_COMPLETE_ARGUMENT_COUNT value unrecognized!"
 #endif
 #endif
+#endif
 
 #if defined(CONFIG_PREEMPT_RT)
 typedef atomic_spinlock_t         nv_spinlock_t;
@@ -1037,7 +1041,7 @@
 #endif
 
 #if !defined(NV_VMWARE)
-#define NV_NUM_PHYSPAGES                num_physpages
+#define NV_NUM_PHYSPAGES                get_num_physpages()
 #define NV_GET_CURRENT_PROCESS()        current->tgid
 #define NV_IN_ATOMIC()                  in_atomic()
 #define NV_LOCAL_BH_DISABLE()           local_bh_disable()
