diff -urN a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2012-06-30 18:13:28.968156002 -0500
+++ b/CMakeLists.txt	2012-06-30 23:58:32.965156002 -0500
@@ -116,7 +116,7 @@
 
 install (
     FILES ${CMAKE_BINARY_DIR}/compiz.pc
-    DESTINATION ${COMPIZ_DESTDIR}${libdir}/pkgconfig
+    DESTINATION ${libdir}/pkgconfig
 )
 
 # Build Google Test and make its headers known
diff -urN a/cmake/CMakeLists.txt b/cmake/CMakeLists.txt
--- a/cmake/CMakeLists.txt	2012-06-30 18:13:28.956156002 -0500
+++ b/cmake/CMakeLists.txt	2012-06-30 18:58:11.995156001 -0500
@@ -25,10 +25,10 @@
 
 install (
     FILES ${_files}
-    DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/compiz/cmake
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/compiz/cmake
 )
 
 install (
     FILES ${_PluginExtensionFiles}
-    DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/compiz/cmake/plugin_extensions
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/compiz/cmake/plugin_extensions
 )
diff -urN a/cmake/CompizGSettings.cmake b/cmake/CompizGSettings.cmake
--- a/cmake/CompizGSettings.cmake	2012-06-30 18:13:28.956156002 -0500
+++ b/cmake/CompizGSettings.cmake	2012-07-01 12:02:45.603156000 -0500
@@ -45,12 +45,12 @@
 		    # if (NOT XDG_INSTALL_PATH)
 		    #	message (\"[WARNING]: Installing GSettings schemas to directory that is not in XDG_DATA_DIRS, you need to add ${COMPIZ_DESTDIR}${_dst} to your XDG_DATA_DIRS in order for GSettings schemas to be found!\"\)
 		    # endif (NOT XDG_INSTALL_PATH)
-		    message (\"-- Installing GSettings schemas to ${COMPIZ_DESTDIR}${_dst}\"\)
-		    file (INSTALL DESTINATION \"${COMPIZ_DESTDIR}${_dst}\"
+		    message (\"-- Installing GSettings schemas to ${_dst}\"\)
+		    file (INSTALL DESTINATION \"${_dst}\"
 			  TYPE FILE
 			  FILES \"${_src}\")
-		    message (\"-- Recompiling GSettings schemas in ${COMPIZ_DESTDIR}${_dst}\"\)
-		    execute_process (COMMAND ${GLIB_COMPILE_SCHEMAS} ${COMPIZ_DESTDIR}${_dst})
+		    message (\"-- Recompiling GSettings schemas in ${_dst}\"\)
+		    execute_process (COMMAND ${GLIB_COMPILE_SCHEMAS} ${_dst})
 		endif (\"$ENV{USER}\" STREQUAL \"root\"\)
 		")
     endif ()
diff -urN a/cmake/CompizGconf.cmake b/cmake/CompizGconf.cmake
--- a/cmake/CompizGconf.cmake	2012-06-30 18:13:28.956156002 -0500
+++ b/cmake/CompizGconf.cmake	2012-07-01 12:56:05.563156008 -0500
@@ -29,7 +29,7 @@
     endif ()
     install (
 	FILES "${_src}"
-	DESTINATION "${COMPIZ_DESTDIR}${_dst}"
+	DESTINATION "${_dst}"
     )
 endfunction ()
 
diff -urN a/cmake/CompizPlugin.cmake b/cmake/CompizPlugin.cmake
--- a/cmake/CompizPlugin.cmake	2012-06-30 18:13:28.956156002 -0500
+++ b/cmake/CompizPlugin.cmake	2012-07-01 00:09:07.097156002 -0500
@@ -247,7 +247,7 @@
 	    # install xml
 	    install (
 		FILES ${_translated_xml}
-		DESTINATION ${COMPIZ_DESTDIR}${PLUGIN_XMLDIR}
+		DESTINATION ${PLUGIN_XMLDIR}
 	    )
 	endif (_translated_xml)
 
@@ -278,11 +278,11 @@
 
 		install (
 		    FILES ${CMAKE_BINARY_DIR}/generated/compiz-${plugin}.pc
-		    DESTINATION ${COMPIZ_DESTDIR}${PLUGIN_PKGDIR}
+		    DESTINATION ${PLUGIN_PKGDIR}
 		)
 		install (
 		    DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/include/${plugin}
-		    DESTINATION ${COMPIZ_DESTDIR}${PLUGIN_INCDIR}/compiz
+		    DESTINATION ${PLUGIN_INCDIR}/compiz
 		)
 	    endif ()
 	endif ()
@@ -413,7 +413,7 @@
 
 	install (
 	    TARGETS ${plugin}
-	    LIBRARY DESTINATION ${COMPIZ_DESTDIR}${PLUGIN_LIBDIR}
+	    LIBRARY DESTINATION ${PLUGIN_LIBDIR}
 	)
 
 	compiz_add_uninstall ()
diff -urN a/cmake/base.cmake b/cmake/base.cmake
--- a/cmake/base.cmake	2012-06-30 18:13:28.956156002 -0500
+++ b/cmake/base.cmake	2012-07-01 12:34:19.341155994 -0500
@@ -45,8 +45,8 @@
         compiz_print_configure_footer ()
     endif ()
     add_custom_target (findcompiz_install
-	${CMAKE_COMMAND} -E make_directory ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules &&
-	${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/cmake/FindCompiz.cmake ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules
+		${CMAKE_COMMAND} -E make_directory ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules &&
+		${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/cmake/FindCompiz.cmake ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules
     )
 endfunction ()
 
diff -urN a/cmake/plugin_extensions/CompizGenGSettings.cmake b/cmake/plugin_extensions/CompizGenGSettings.cmake
--- a/cmake/plugin_extensions/CompizGenGSettings.cmake	2012-06-30 18:13:28.956156002 -0500
+++ b/cmake/plugin_extensions/CompizGenGSettings.cmake	2012-07-01 13:00:22.691155998 -0500
@@ -74,7 +74,7 @@
 		    message (\"[WARNING]: Installing GSettings schemas to a custom location that might not be in XDG_DATA_DIRS, you need to add ${COMPIZ_DESTDIR}${_dst} to your XDG_DATA_DIRS in order for GSettings schemas to be found!\"\)
 		    # endif (NOT XDG_INSTALL_PATH)
 		    message (\"-- Installing GSettings schemas to ${COMPIZ_DESTDIR}${_dst}\"\)
-		    file (INSTALL DESTINATION \"${COMPIZ_DESTDIR}${_dst}\"
+		    file (INSTALL DESTINATION \"${_dst}\"
 			  TYPE FILE
 			  FILES \"${_src}\")
 		    message (\"-- Recompiling GSettings schemas in ${COMPIZ_DESTDIR}${_dst}\"\)
diff -urN a/cmake/plugin_extensions/CompizGenGconf.cmake b/cmake/plugin_extensions/CompizGenGconf.cmake
--- a/cmake/plugin_extensions/CompizGenGconf.cmake	2012-06-30 18:13:28.956156002 -0500
+++ b/cmake/plugin_extensions/CompizGenGconf.cmake	2012-07-01 00:15:50.323156004 -0500
@@ -55,7 +55,7 @@
     endif ()
     install (
 	FILES "${_src}"
-	DESTINATION "${COMPIZ_DESTDIR}${_dst}"
+	DESTINATION "${_dst}"
     )
 endfunction ()
 
diff -urN a/cmake/plugin_extensions/CompizGenInstallImages.cmake b/cmake/plugin_extensions/CompizGenInstallImages.cmake
--- a/cmake/plugin_extensions/CompizGenInstallImages.cmake	2012-06-30 18:13:28.956156002 -0500
+++ b/cmake/plugin_extensions/CompizGenInstallImages.cmake	2012-07-01 12:12:17.795156003 -0500
@@ -19,7 +19,7 @@
     compiz_images_prepare_dirs ()
     install (
 	DIRECTORY ${CMAKE_CURRENT_SOURCE_DIR}/images
-	DESTINATION ${COMPIZ_DESTDIR}${PLUGIN_IMAGEDIR}
+	DESTINATION ${PLUGIN_IMAGEDIR}
     )
     list (APPEND COMPIZ_DEFINITIONS_ADD "-DIMAGEDIR='\"${PLUGIN_IMAGEDIR}\"'")
 endif ()
diff -urN a/gtk/gnome/CMakeLists.txt b/gtk/gnome/CMakeLists.txt
--- a/gtk/gnome/CMakeLists.txt	2012-06-30 18:13:28.957156002 -0500
+++ b/gtk/gnome/CMakeLists.txt	2012-06-30 18:52:30.697156001 -0500
@@ -6,7 +6,7 @@
     )
     install (
 	FILES ${desktop_file}
-	DESTINATION ${COMPIZ_DESTDIR}${datadir}/applications
+	DESTINATION ${datadir}/applications
     )
 
     add_custom_target (compiz.desktop ALL SOURCES ${desktop_file})
diff -urN a/gtk/window-decorator/CMakeLists.txt b/gtk/window-decorator/CMakeLists.txt
--- a/gtk/window-decorator/CMakeLists.txt	2012-06-30 18:13:28.957156002 -0500
+++ b/gtk/window-decorator/CMakeLists.txt	2012-07-01 00:17:37.635156006 -0500
@@ -29,7 +29,7 @@
 	    ${CMAKE_CURRENT_SOURCE_DIR}/gwd.schemas.in
 	    ${gwd_schema}
 	)
-	compiz_install_gconf_schema (${CMAKE_CURRENT_BINARY_DIR}/gwd.schemas ${SCHEMADIR})
+compiz_install_gconf_schema (${CMAKE_CURRENT_BINARY_DIR}/gwd.schemas ${SCHEMADIR})
     endif (USE_GCONF)
 
     add_executable (gtk-window-decorator
@@ -71,7 +71,7 @@
 
     install (
 	TARGETS gtk-window-decorator
-	DESTINATION ${COMPIZ_DESTDIR}${exec_prefix}
+	DESTINATION ${exec_prefix}
     )
 
 
diff -urN a/images/CMakeLists.txt b/images/CMakeLists.txt
--- a/images/CMakeLists.txt	2012-06-30 18:13:28.958156002 -0500
+++ b/images/CMakeLists.txt	2012-07-01 00:19:12.741155979 -0500
@@ -5,5 +5,5 @@
 
 install (
     FILES ${_images}
-    DESTINATION ${COMPIZ_DESTDIR}${compiz_sharedir}
+    DESTINATION ${compiz_sharedir}
 )
diff -urN a/include/CMakeLists.txt b/include/CMakeLists.txt
--- a/include/CMakeLists.txt	2012-06-30 18:13:28.969156002 -0500
+++ b/include/CMakeLists.txt	2012-07-01 00:19:57.861156007 -0500
@@ -4,7 +4,7 @@
 
 install (
     FILES ${_headers}
-    DESTINATION ${COMPIZ_DESTDIR}${includedir}/compiz
+    DESTINATION ${includedir}/compiz
 )
 
 add_subdirectory (core)
diff -urN a/kde/window-decorator-kde4/CMakeLists.txt b/kde/window-decorator-kde4/CMakeLists.txt
--- a/kde/window-decorator-kde4/CMakeLists.txt	2012-06-30 18:13:28.969156002 -0500
+++ b/kde/window-decorator-kde4/CMakeLists.txt	2012-07-01 00:20:06.749156002 -0500
@@ -66,7 +66,7 @@
 
 	install (
 	    TARGETS kde4-window-decorator
-	    DESTINATION ${COMPIZ_DESTDIR}${exec_prefix}
+	    DESTINATION ${exec_prefix}
 	)
 
     else (KDE4_FOUND)
diff -urN a/libdecoration/CMakeLists.txt b/libdecoration/CMakeLists.txt
--- a/libdecoration/CMakeLists.txt	2012-06-30 18:13:28.967156002 -0500
+++ b/libdecoration/CMakeLists.txt	2012-07-01 00:18:54.708156005 -0500
@@ -35,7 +35,7 @@
 
 install (
     TARGETS decoration
-    DESTINATION ${COMPIZ_DESTDIR}${libdir}
+    DESTINATION ${libdir}
 )
 
 set (DECORATION_LIBS "")
@@ -55,5 +55,5 @@
 
 install (
     FILES ${CMAKE_CURRENT_BINARY_DIR}/libdecoration.pc
-    DESTINATION ${COMPIZ_DESTDIR}${libdir}/pkgconfig
+    DESTINATION ${libdir}/pkgconfig
 )
diff -urN a/metadata/CMakeLists.txt b/metadata/CMakeLists.txt
--- a/metadata/CMakeLists.txt	2012-06-30 18:13:28.967156002 -0500
+++ b/metadata/CMakeLists.txt	2012-07-01 00:19:43.973156001 -0500
@@ -33,7 +33,7 @@
 
 install (
     FILES ${_core_xml_file}
-    DESTINATION ${COMPIZ_DESTDIR}${compiz_metadatadir}
+    DESTINATION ${compiz_metadatadir}
 )
 
 add_custom_target (core-xml-file ALL
diff -urN a/po/CMakeLists.txt b/po/CMakeLists.txt
--- a/po/CMakeLists.txt	2012-06-30 18:13:28.974156002 -0500
+++ b/po/CMakeLists.txt	2012-06-30 18:35:13.249156001 -0500
@@ -45,7 +45,7 @@
 
     install (
 	FILES ${po_BINARY_DIR}/${catalog}/LC_MESSAGES/compiz.mo
-	DESTINATION ${COMPIZ_DESTDIR}${datadir}/locale/${catalog}/LC_MESSAGES
+	DESTINATION ${datadir}/locale/${catalog}/LC_MESSAGES
     )
 
     set (compiz_catalog_targets ${compiz_catalog_targets} ${po_BINARY_DIR}/${catalog}/LC_MESSAGES/compiz.mo)
diff -urN a/src/CMakeLists.txt b/src/CMakeLists.txt
--- a/src/CMakeLists.txt	2012-06-30 18:13:28.959156002 -0500
+++ b/src/CMakeLists.txt	2012-07-01 00:18:26.772155925 -0500
@@ -178,12 +178,12 @@
 
 install (
     TARGETS compiz_core
-  	LIBRARY DESTINATION ${COMPIZ_DESTDIR}${libdir}
+  	LIBRARY DESTINATION ${libdir}
 )
 
 install (
     TARGETS compiz
-    RUNTIME DESTINATION ${COMPIZ_DESTDIR}${exec_prefix}
+    RUNTIME DESTINATION ${exec_prefix}
 )
 
 enable_coverage_report( TARGETS compiz )
diff -urN a/xslt/CMakeLists.txt b/xslt/CMakeLists.txt
--- a/xslt/CMakeLists.txt	2012-06-30 18:13:28.968156002 -0500
+++ b/xslt/CMakeLists.txt	2012-06-30 18:58:33.483156001 -0500
@@ -16,5 +16,5 @@
 
 install (
     FILES ${_files}
-    DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/compiz/xslt
+    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/compiz/xslt
 )
