diff -urN a/CMakeLists.txt b/CMakeLists.txt
--- a/CMakeLists.txt	2012-06-28 18:51:14.737892849 -0500
+++ b/CMakeLists.txt	2012-06-28 19:54:32.611516021 -0500
@@ -120,7 +120,7 @@
 
 install (
     FILES ${CMAKE_BINARY_DIR}/compiz.pc
-    DESTINATION ${libdir}/pkgconfig
+    DESTINATION ${COMPIZ_DESTDIR}${libdir}/pkgconfig
 )
 
 # Build Google Test and make its headers known
diff -urN a/cmake/CMakeLists.txt b/cmake/CMakeLists.txt
--- a/cmake/CMakeLists.txt	2012-06-28 18:51:14.735892871 -0500
+++ b/cmake/CMakeLists.txt	2012-06-28 21:54:05.347740549 -0500
@@ -25,10 +25,10 @@
 
 install (
     FILES ${_files}
-    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/compiz/cmake
+    DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/compiz/cmake
 )
 
 install (
     FILES ${_PluginExtensionFiles}
-    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/compiz/cmake/plugin_extensions
+    DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/compiz/cmake/plugin_extensions
 )
diff -urN a/cmake/base.cmake b/cmake/base.cmake
--- a/cmake/base.cmake	2012-06-28 18:51:14.735892871 -0500
+++ b/cmake/base.cmake	2012-06-28 22:32:52.865781532 -0500
@@ -7,9 +7,9 @@
     mark_as_advanced (FORCE GCONFTOOL_EXECUTABLE)
 
     if (NOT COMPIZ_INSTALL_GCONF_SCHEMA_DIR)
-        set (SCHEMADIR "${CMAKE_INSTALL_PREFIX}/share/gconf/schemas")
+        set (SCHEMADIR "${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/gconf/schemas")
     else (NOT COMPIZ_INSTALL_GCONF_SCHEMA_DIR)
-        set (SCHEMADIR "${COMPIZ_INSTALL_GCONF_SCHEMA_DIR}")
+        set (SCHEMADIR "${COMPIZ_DESTDIR}${COMPIZ_INSTALL_GCONF_SCHEMA_DIR}")
     endif (NOT COMPIZ_INSTALL_GCONF_SCHEMA_DIR)
 
     if (NOT GCONF_FOUND OR NOT GCONFTOOL_EXECUTABLE)
@@ -51,10 +51,10 @@
     install (FILES
 	     ${CMAKE_CURRENT_SOURCE_DIR}/cmake/FindCompiz.cmake
 	     DESTINATION
-	     ${CMAKE_INSTALL_PREFIX}/share/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/)
+	     ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/)
     add_custom_target (findcompiz_install
-	${CMAKE_COMMAND} -E make_directory ${CMAKE_ROOT}/Modules &&
-	${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/cmake/FindCompiz.cmake ${CMAKE_ROOT}/Modules
+	${CMAKE_COMMAND} -E make_directory ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules &&
+	${CMAKE_COMMAND} -E copy ${CMAKE_SOURCE_DIR}/cmake/FindCompiz.cmake ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules
     )
     find_file (_find_compizconfig FindCompizConfig.cmake PATHS ${CMAKE_ROOT}/Modules ${ARGN})
     if (NOT _find_compizconfig)
@@ -67,11 +67,11 @@
     install (FILES
 	     ${CMAKE_CURRENT_SOURCE_DIR}/compizconfig/libcompizconfig/cmake/FindCompizConfig.cmake
 	     DESTINATION
-	     ${CMAKE_INSTALL_PREFIX}/share/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/)
+	     ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/cmake-${CMAKE_MAJOR_VERSION}.${CMAKE_MINOR_VERSION}/)
     add_custom_target (
 	findcompizconfig_install
-	${CMAKE_COMMAND} -E make_directory ${CMAKE_ROOT}/Modules &&
-	${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/compizconfig/libcompizconfig/cmake/FindCompizConfig.cmake ${CMAKE_ROOT}/Modules
+	${CMAKE_COMMAND} -E make_directory ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules &&
+	${CMAKE_COMMAND} -E copy ${CMAKE_CURRENT_SOURCE_DIR}/compizconfig/libcompizconfig/cmake/FindCompizConfig.cmake ${COMPIZ_DESTDIR}${CMAKE_ROOT}/Modules
     )
 endfunction ()
 
diff -urN a/compizconfig/libcompizconfig/CMakeLists.txt b/compizconfig/libcompizconfig/CMakeLists.txt
--- a/compizconfig/libcompizconfig/CMakeLists.txt	2012-06-28 18:51:14.744892772 -0500
+++ b/compizconfig/libcompizconfig/CMakeLists.txt	2012-06-28 21:55:21.439761488 -0500
@@ -140,7 +140,7 @@
 
 install (
     FILES ${CMAKE_CURRENT_BINARY_DIR}/libcompizconfig.pc
-    DESTINATION ${libdir}/pkgconfig
+    DESTINATION ${COMPIZ_DESTDIR}${libdir}/pkgconfig
 )
 
 
diff -urN a/compizconfig/libcompizconfig/cmake/CMakeLists.txt b/compizconfig/libcompizconfig/cmake/CMakeLists.txt
--- a/compizconfig/libcompizconfig/cmake/CMakeLists.txt	2012-06-28 18:51:14.744892772 -0500
+++ b/compizconfig/libcompizconfig/cmake/CMakeLists.txt	2012-06-28 21:54:54.097113229 -0500
@@ -4,5 +4,5 @@
 
 install (
     FILES ${_files}
-    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/compiz/cmake
+    DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/compiz/cmake
 )
diff -urN a/compizconfig/libcompizconfig/include/CMakeLists.txt b/compizconfig/libcompizconfig/include/CMakeLists.txt
--- a/compizconfig/libcompizconfig/include/CMakeLists.txt	2012-06-28 18:51:14.744892772 -0500
+++ b/compizconfig/libcompizconfig/include/CMakeLists.txt	2012-06-28 21:55:12.737873422 -0500
@@ -5,5 +5,5 @@
 
 install (
 	FILES ${_headers}
-	DESTINATION ${libcompizconfig_includedir}
+	DESTINATION ${COMPIZ_DESTDIR}${libcompizconfig_includedir}
 	)
diff -urN a/compizconfig/libcompizconfig/src/CMakeLists.txt b/compizconfig/libcompizconfig/src/CMakeLists.txt
--- a/compizconfig/libcompizconfig/src/CMakeLists.txt	2012-06-28 18:51:14.744892772 -0500
+++ b/compizconfig/libcompizconfig/src/CMakeLists.txt	2012-06-28 21:55:29.097662993 -0500
@@ -76,5 +76,5 @@
 
 install (
     TARGETS compizconfig
-    DESTINATION ${libcompizconfig_libdir}
+    DESTINATION ${COMPIZ_DESTDIR}${libcompizconfig_libdir}
 )
diff -urN a/gtk/gnome/CMakeLists.txt b/gtk/gnome/CMakeLists.txt
--- a/gtk/gnome/CMakeLists.txt	2012-06-28 18:51:14.792892242 -0500
+++ b/gtk/gnome/CMakeLists.txt	2012-06-28 21:55:38.875537235 -0500
@@ -6,7 +6,7 @@
     )
     install (
 	FILES ${desktop_file}
-	DESTINATION ${datadir}/applications
+	DESTINATION ${COMPIZ_DESTDIR}${datadir}/applications
     )
 
     add_custom_target (compiz.desktop ALL SOURCES ${desktop_file})
diff -urN a/gtk/window-decorator/CMakeLists.txt b/gtk/window-decorator/CMakeLists.txt
--- a/gtk/window-decorator/CMakeLists.txt	2012-06-28 18:51:14.791892253 -0500
+++ b/gtk/window-decorator/CMakeLists.txt	2012-06-28 21:55:33.575605396 -0500
@@ -71,7 +71,7 @@
 
     install (
 	TARGETS gtk-window-decorator
-	DESTINATION ${exec_prefix}
+	DESTINATION ${COMPIZ_DESTDIR}${exec_prefix}
     )
 
 
diff -urN a/images/CMakeLists.txt b/images/CMakeLists.txt
--- a/images/CMakeLists.txt	2012-06-28 18:51:14.735892871 -0500
+++ b/images/CMakeLists.txt	2012-06-28 21:55:46.083444535 -0500
@@ -5,5 +5,5 @@
 
 install (
     FILES ${_images}
-    DESTINATION ${compiz_sharedir}
+    DESTINATION ${COMPIZ_DESTDIR}${compiz_sharedir}
 )
diff -urN a/include/CMakeLists.txt b/include/CMakeLists.txt
--- a/include/CMakeLists.txt	2012-06-28 18:51:14.736892860 -0500
+++ b/include/CMakeLists.txt	2012-06-28 21:55:50.242391054 -0500
@@ -4,7 +4,7 @@
 
 install (
     FILES ${_headers}
-    DESTINATION ${includedir}/compiz
+    DESTINATION ${COMPIZ_DESTDIR}${includedir}/compiz
 )
 
 add_subdirectory (core)
diff -urN a/include/core/CMakeLists.txt b/include/core/CMakeLists.txt
--- a/include/core/CMakeLists.txt	2012-06-28 18:51:14.736892860 -0500
+++ b/include/core/CMakeLists.txt	2012-06-28 18:58:50.338834177 -0500
@@ -22,4 +22,4 @@
     abiversion.h
 )
 
-install (FILES ${_headers} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${_headers} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
diff -urN a/kde/window-decorator-kde4/CMakeLists.txt b/kde/window-decorator-kde4/CMakeLists.txt
--- a/kde/window-decorator-kde4/CMakeLists.txt	2012-06-28 18:51:14.737892849 -0500
+++ b/kde/window-decorator-kde4/CMakeLists.txt	2012-06-28 21:55:56.303313113 -0500
@@ -66,7 +66,7 @@
 
 	install (
 	    TARGETS kde4-window-decorator
-	    DESTINATION ${exec_prefix}
+	    DESTINATION ${COMPIZ_DESTDIR}${exec_prefix}
 	)
 
     else (KDE4_FOUND)
diff -urN a/libdecoration/CMakeLists.txt b/libdecoration/CMakeLists.txt
--- a/libdecoration/CMakeLists.txt	2012-06-28 18:51:14.735892871 -0500
+++ b/libdecoration/CMakeLists.txt	2012-06-28 21:56:13.930086465 -0500
@@ -35,7 +35,7 @@
 
 install (
     TARGETS decoration
-    DESTINATION ${libdir}
+    DESTINATION ${COMPIZ_DESTDIR}${libdir}
 )
 
 set (DECORATION_LIBS "")
@@ -55,5 +55,5 @@
 
 install (
     FILES ${CMAKE_CURRENT_BINARY_DIR}/libdecoration.pc
-    DESTINATION ${libdir}/pkgconfig
+    DESTINATION ${COMPIZ_DESTDIR}${libdir}/pkgconfig
 )
diff -urN a/metadata/CMakeLists.txt b/metadata/CMakeLists.txt
--- a/metadata/CMakeLists.txt	2012-06-28 18:51:14.736892860 -0500
+++ b/metadata/CMakeLists.txt	2012-06-28 21:56:18.282030514 -0500
@@ -33,7 +33,7 @@
 
 install (
     FILES ${_core_xml_file}
-    DESTINATION ${compiz_metadatadir}
+    DESTINATION ${COMPIZ_DESTDIR}${compiz_metadatadir}
 )
 
 add_custom_target (core-xml-file ALL
diff -urN a/plugins/decor/CMakeLists.txt b/plugins/decor/CMakeLists.txt
--- a/plugins/decor/CMakeLists.txt	2012-06-28 18:51:14.817891966 -0500
+++ b/plugins/decor/CMakeLists.txt	2012-06-28 21:56:21.780985528 -0500
@@ -19,4 +19,4 @@
     )
 endif (COMPIZ_BUILD_WITH_RPATH AND NOT COMPIZ_DISABLE_PLUGIN_DECOR)
 
-install (PROGRAMS "src/compiz-decorator" DESTINATION ${exec_prefix})
+install (PROGRAMS "src/compiz-decorator" DESTINATION ${COMPIZ_DESTDIR}${exec_prefix})
diff -urN a/po/CMakeLists.txt b/po/CMakeLists.txt
--- a/po/CMakeLists.txt	2012-06-28 18:51:14.758892617 -0500
+++ b/po/CMakeLists.txt	2012-06-28 21:56:26.209928587 -0500
@@ -45,7 +45,7 @@
 
     install (
 	FILES ${po_BINARY_DIR}/${catalog}/LC_MESSAGES/compiz.mo
-	DESTINATION ${datadir}/locale/${catalog}/LC_MESSAGES
+	DESTINATION ${COMPIZ_DESTDIR}${datadir}/locale/${catalog}/LC_MESSAGES
     )
 
     set (compiz_catalog_targets ${compiz_catalog_targets} ${po_BINARY_DIR}/${catalog}/LC_MESSAGES/compiz.mo)
diff -urN a/src/CMakeLists.txt b/src/CMakeLists.txt
--- a/src/CMakeLists.txt	2012-06-28 18:51:14.739892827 -0500
+++ b/src/CMakeLists.txt	2012-06-28 21:56:34.947816259 -0500
@@ -186,12 +186,12 @@
 
 install (
     TARGETS compiz_core
-  	LIBRARY DESTINATION ${libdir}
+  	LIBRARY DESTINATION ${COMPIZ_DESTDIR}${libdir}
 )
 
 install (
     TARGETS compiz
-    RUNTIME DESTINATION ${exec_prefix}
+    RUNTIME DESTINATION ${COMPIZ_DESTDIR}${exec_prefix}
 )
 
 enable_coverage_report( TARGETS compiz )
diff -urN a/src/logmessage/CMakeLists.txt b/src/logmessage/CMakeLists.txt
--- a/src/logmessage/CMakeLists.txt	2012-06-28 18:51:14.742892794 -0500
+++ b/src/logmessage/CMakeLists.txt	2012-06-28 18:58:50.351834024 -0500
@@ -30,4 +30,4 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
diff -urN a/src/pluginclasshandler/CMakeLists.txt b/src/pluginclasshandler/CMakeLists.txt
--- a/src/pluginclasshandler/CMakeLists.txt	2012-06-28 18:51:14.743892783 -0500
+++ b/src/pluginclasshandler/CMakeLists.txt	2012-06-28 18:58:50.354833990 -0500
@@ -43,4 +43,4 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
diff -urN a/src/point/CMakeLists.txt b/src/point/CMakeLists.txt
--- a/src/point/CMakeLists.txt	2012-06-28 18:51:14.741892805 -0500
+++ b/src/point/CMakeLists.txt	2012-06-28 18:58:50.350834036 -0500
@@ -52,7 +52,7 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
 
 TARGET_LINK_LIBRARIES( 
   compiz_point
diff -urN a/src/rect/CMakeLists.txt b/src/rect/CMakeLists.txt
--- a/src/rect/CMakeLists.txt	2012-06-28 18:51:14.738892838 -0500
+++ b/src/rect/CMakeLists.txt	2012-06-28 18:58:50.340834153 -0500
@@ -50,7 +50,7 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
 
 TARGET_LINK_LIBRARIES( 
   compiz_rect
diff -urN a/src/region/CMakeLists.txt b/src/region/CMakeLists.txt
--- a/src/region/CMakeLists.txt	2012-06-28 18:51:14.740892816 -0500
+++ b/src/region/CMakeLists.txt	2012-06-28 18:58:50.346834084 -0500
@@ -53,7 +53,7 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
 
 TARGET_LINK_LIBRARIES( 
   compiz_region
diff -urN a/src/servergrab/CMakeLists.txt b/src/servergrab/CMakeLists.txt
--- a/src/servergrab/CMakeLists.txt	2012-06-28 18:51:14.740892816 -0500
+++ b/src/servergrab/CMakeLists.txt	2012-06-28 18:58:50.348834060 -0500
@@ -39,4 +39,4 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
diff -urN a/src/string/CMakeLists.txt b/src/string/CMakeLists.txt
--- a/src/string/CMakeLists.txt	2012-06-28 18:51:14.741892805 -0500
+++ b/src/string/CMakeLists.txt	2012-06-28 18:58:50.349834048 -0500
@@ -28,4 +28,4 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
diff -urN a/src/timer/CMakeLists.txt b/src/timer/CMakeLists.txt
--- a/src/timer/CMakeLists.txt	2012-06-28 18:51:14.742892794 -0500
+++ b/src/timer/CMakeLists.txt	2012-06-28 18:58:50.352834012 -0500
@@ -55,7 +55,7 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
 
 TARGET_LINK_LIBRARIES( 
   compiz_timer
diff -urN a/src/window/constrainment/CMakeLists.txt b/src/window/constrainment/CMakeLists.txt
--- a/src/window/constrainment/CMakeLists.txt	2012-06-28 18:51:14.738892838 -0500
+++ b/src/window/constrainment/CMakeLists.txt	2012-06-28 18:58:50.341834141 -0500
@@ -53,7 +53,7 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
 
 TARGET_LINK_LIBRARIES( 
   compiz_window_constrainment
diff -urN a/src/window/extents/CMakeLists.txt b/src/window/extents/CMakeLists.txt
--- a/src/window/extents/CMakeLists.txt	2012-06-28 18:51:14.739892827 -0500
+++ b/src/window/extents/CMakeLists.txt	2012-06-28 18:58:50.345834095 -0500
@@ -51,7 +51,7 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
 
 TARGET_LINK_LIBRARIES( 
   compiz_window_extents
diff -urN a/src/window/geometry/CMakeLists.txt b/src/window/geometry/CMakeLists.txt
--- a/src/window/geometry/CMakeLists.txt	2012-06-28 18:51:14.738892838 -0500
+++ b/src/window/geometry/CMakeLists.txt	2012-06-28 18:58:50.343834117 -0500
@@ -51,7 +51,7 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
 
 TARGET_LINK_LIBRARIES( 
   compiz_window_geometry
diff -urN a/src/window/geometry-saver/CMakeLists.txt b/src/window/geometry-saver/CMakeLists.txt
--- a/src/window/geometry-saver/CMakeLists.txt	2012-06-28 18:51:14.739892827 -0500
+++ b/src/window/geometry-saver/CMakeLists.txt	2012-06-28 18:58:50.344834106 -0500
@@ -53,7 +53,7 @@
   PUBLIC_HEADER "${PUBLIC_HEADERS}"
 )
 
-install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_CORE_INCLUDE_DIR})
+install (FILES ${PUBLIC_HEADERS} DESTINATION ${COMPIZ_DESTDIR}${COMPIZ_CORE_INCLUDE_DIR})
 
 TARGET_LINK_LIBRARIES ( 
   compiz_window_geometry_saver
diff -urN a/xslt/CMakeLists.txt b/xslt/CMakeLists.txt
--- a/xslt/CMakeLists.txt	2012-06-28 18:51:14.763892562 -0500
+++ b/xslt/CMakeLists.txt	2012-06-28 21:56:38.993764250 -0500
@@ -16,5 +16,5 @@
 
 install (
     FILES ${_files}
-    DESTINATION ${CMAKE_INSTALL_PREFIX}/share/compiz/xslt
+    DESTINATION ${COMPIZ_DESTDIR}${CMAKE_INSTALL_PREFIX}/share/compiz/xslt
 )
