diff -urN a/compizconfig/cmake/exec_setup_py_with_destdir.cmake b/compizconfig/cmake/exec_setup_py_with_destdir.cmake
--- a/compizconfig/cmake/exec_setup_py_with_destdir.cmake	2012-06-29 02:33:48.179800999 -0500
+++ b/compizconfig/cmake/exec_setup_py_with_destdir.cmake	2012-06-30 10:16:47.300740001 -0500
@@ -15,10 +15,10 @@
 
 cmake_policy (SET CMP0012 NEW)
 
-message (${BUILD_DEB})
+message (STATUS ${BUILD_DEB})
 
 if (BUILD_DEB)
-    message ("Installing Debian Format")
+	message (STATUS "Installing Debian Format")
     set (EXTRAARGS "--install-layout=deb")
 endif (BUILD_DEB)
 
