diff -urN a/virtinst/OSDistro.py b/virtinst/OSDistro.py
--- a/virtinst/OSDistro.py	2012-01-24 18:00:05.000000000 -0600
+++ b/virtinst/OSDistro.py	2012-02-02 13:17:53.539111296 -0600
@@ -896,12 +896,10 @@
 
     def __init__(self, uri, arch, vmtype=None, scratchdir=None):
         Distro.__init__(self, uri, arch, vmtype, scratchdir)
-        if uri.count("installer-i386"):
-            self._treeArch = "i386"
-        elif uri.count("installer-amd64"):
+        if arch == "x86_64":
             self._treeArch = "amd64"
         else:
-            self._treeArch = "i386"
+            self._treeArch = arch
 
         if re.match(r'i[4-9]86', arch):
             self.arch = 'i386'
