diff -urNp a/drivers/bluetooth/btusb.c b/drivers/bluetooth/btusb.c
--- a/drivers/bluetooth/btusb.c	2012-07-08 22:42:40.731685786 -0500
+++ b/drivers/bluetooth/btusb.c	2012-07-08 22:58:06.864680304 -0500
@@ -81,6 +81,9 @@ static struct usb_device_id btusb_table[
 	/* Apple MacBookPro8,2 */
 	{ USB_DEVICE(0x05ac, 0x821a) },
 
+	/* Apple MacBookPro10,1 */
+	{ USB_DEVICE(0x05ac, 0x8286) },
+
 	/* Apple MacMini5,1 */
 	{ USB_DEVICE(0x05ac, 0x8281) },
 
