diff -urN a/blueman/Functions.py b/blueman/Functions.py
--- a/blueman/Functions.py	2011-10-08 12:09:16.000000000 -0700
+++ b/blueman/Functions.py	2013-04-09 15:56:36.084475469 -0700
@@ -190,7 +190,7 @@
 	if icon.props.height > size:
 		new_w = int(size * ( float(icon.props.height) / icon.props.width ))
 		new_h = size
-		icon = icon.scale_simple(dest_width, dest_height, gtk.gdk.INTERP_BILINEAR)
+		icon = icon.scale_simple(new_w, new_h, gtk.gdk.INTERP_BILINEAR)
 	
 	return icon
 	
