diff -urN a/Makefile b/Makefile
--- a/Makefile	2008-05-01 07:03:15.000000000 -0500
+++ b/Makefile	2011-11-22 09:28:25.676286049 -0600
@@ -1,4 +1,4 @@
-DIR=/usr/local
+DIR=${DESTDIR}/usr
 
 all:
 	@echo "Available actions in this makefile: install, uninstall"
diff -urN a/src/compiz-switch b/src/compiz-switch
--- a/src/compiz-switch	2008-05-01 07:06:25.000000000 -0500
+++ b/src/compiz-switch	2011-11-22 09:09:46.947286002 -0600
@@ -12,7 +12,7 @@
 # of MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
 # See the GNU General Public License for more details.
 
-COMPIZ_WRAPPER="/usr/local/bin/compiz-wrapper"
+COMPIZ_WRAPPER="/usr/bin/compiz-wrapper"
 COMPIZ_BIN=compiz
 
 VERSION=0.4.0~source
@@ -51,7 +51,7 @@
   SWITCH_TO_WM=xfwm4
 fi
 
-PIXDIR="/usr/local/share/pixmaps"
+PIXDIR="/usr/share/pixmaps"
 CHANGE_ICON="$HOME/.icons/compiz-switch.png"
 
 while [ $# -gt 0 ]
diff -urN a/src/compiz-wrapper b/src/compiz-wrapper
--- a/src/compiz-wrapper	2008-05-01 07:05:51.000000000 -0500
+++ b/src/compiz-wrapper	2011-11-22 09:09:52.901286004 -0600
@@ -34,8 +34,8 @@
 # 
 
 
-COMPIZ_BIN_PATH="/usr/local/bin/" # For window decorators and compiz
-PLUGIN_PATH="/usr/local/lib/compiz/" 
+COMPIZ_BIN_PATH="/usr/bin/" # For window decorators and compiz
+PLUGIN_PATH="/usr/lib/compiz/" 
 GLXINFO="/usr/bin/glxinfo"
 COMPIZ_NAME="compiz" # Final name for compiz (compiz.real) 
 
