Common subdirectories: rubyripper-0.6.0/locale and rubyripper-0.6.0-fm-titles/locale
diff -u rubyripper-0.6.0/rubyripper_gtk2.rb rubyripper-0.6.0-fm-titles/rubyripper_gtk2.rb
--- rubyripper-0.6.0/rubyripper_gtk2.rb	2010-06-18 14:36:16.000000000 -0500
+++ rubyripper-0.6.0-fm-titles/rubyripper_gtk2.rb	2011-05-03 11:34:29.068695955 -0500
@@ -142,6 +142,10 @@
 			@buttonicons[4].stock = Gtk::Stock::CANCEL
 		end
 
+		if @current_instance == "GtkMetadata"
+			@instances['GtkMetadata'].setTracksEnabled(!@settings['image'])
+		end
+
 		object.display.show_all()
 	end
 	
@@ -513,6 +517,15 @@
 		end
 	end
 
+	def setTracksEnabled(sensitive)
+		if !sensitive
+			@allTracksButton.active = true
+			@checkTrackArray.each{|checkbox| checkbox.active = true}
+		end
+		@allTracksButton.sensitive = sensitive
+		@checkTrackArray.each{|checkbox| checkbox.sensitive = sensitive}
+	end
+
 	def configTrackValues(update = false)
 		if !update
 			@table20.column_spacings = 5
@@ -668,10 +681,14 @@
 		
 		if image
 		    @tracks_to_rip = ["image"]
+
+			@cd.audiotracks.times do |index|
+				@cd.md.tracklist[index] = @trackEntryArray[index].text
+			end
 		else
 		    @cd.audiotracks.times do |index|
-			@cd.md.tracklist[index] = @trackEntryArray[index].text
-			if @checkTrackArray[index].active? ; @tracks_to_rip << index + 1 end
+				@cd.md.tracklist[index] = @trackEntryArray[index].text
+				if @checkTrackArray[index].active? ; @tracks_to_rip << index + 1 end
 		    end
 		end
 		
