Common subdirectories: rubyripper-0.6.0/locale and rubyripper-0.6.0-fm-nodigest/locale
diff -u rubyripper-0.6.0/rr_lib.rb rubyripper-0.6.0-fm-nodigest/rr_lib.rb
--- rubyripper-0.6.0/rr_lib.rb	2010-06-18 14:36:16.000000000 -0500
+++ rubyripper-0.6.0-fm-nodigest/rr_lib.rb	2011-05-03 13:33:59.459564424 -0500
@@ -1882,7 +1882,9 @@
 			end 
 		end
 		
-		getDigest(track) # Get a MD5-digest for the logfile
+		if !@settings['no_log'] or @errors.size > 0
+			getDigest(track) # Get a MD5-digest for the logfile
+		end
 		@progress += @settings['percentages'][track]
 		@settings['log'].ripPerc(@progress)
 		return true
