diff -u -r cpufreqd-2.4.2/src/cpufreqd_acpi_battery.c cpufreqd-2.4.2-r1/src/cpufreqd_acpi_battery.c
--- cpufreqd-2.4.2/src/cpufreqd_acpi_battery.c	2010-03-28 06:34:54.000000000 -0500
+++ cpufreqd-2.4.2-r1/src/cpufreqd_acpi_battery.c	2010-11-15 20:16:25.997999993 -0600
@@ -35,7 +35,7 @@
 #define CHARGE_NOW	"charge_now"
 #define PRESENT		"present"
 #define STATUS		"status"
-#define CURRENT_NOW	"current_now"
+#define CURRENT_NOW	"power_now"
 
 struct battery_info {
 	int capacity;
