diff -urN a/wminput/plugins/led/Makefile.in b/wminput/plugins/led/Makefile.in
--- a/wminput/plugins/led/Makefile.in	2014-04-27 21:09:48.501854374 -0700
+++ b/wminput/plugins/led/Makefile.in	2014-04-27 21:11:15.225779806 -0700
@@ -5,6 +5,7 @@
 PLUGIN_NAME = led
 SOURCES = led.c
 CFLAGS += -I@top_builddir@/wminput -I@top_builddir@/libcwiid
+LDLIBS += -lm
 INST_DIR = $(CWIID_PLUGINS_DIR)
 
 include $(COMMON)/include/plugin.mak
